### DISCUSSION TITLE
I need help making my first contribution to an open source project

### DISCUSSION THREAD

**Discussion by tomkoevoets** [twbs/bootstrap#39973] [General] [answered]
hey there,

I'm a first year student in it and need to make a small and  bigger contribution to an open source project.
is there anybody that can help me on my way with this since its my first time and I'm reasonably unexperienced in programming.

thanks in advance,

Tom

**Comment by coliff** [comment, marked as answer]
There's lots of guides out there for this kind of thing.

Maybe a good starting point could be: https://goodfirstissue.dev/ or https://www.firsttimersonly.com/ ?

Good luck! 👍
